views/mixins/controls.pug

```diff
@@ -40,6 +40,8 @@ mixin renderPropertyListValue(value)
       td= _.first(value)
   else if /^https?:\/\//.test(value)
     td: a(href=value)= value
+  else if /^adt:\/\//.test(value)
```

Comment:
Can we make this check more generic?

anything with `://` can be treated as hyperlink.

adt is more SAP specific.